### ISSUE TITLE
Spoof dpi - Stopped working in Russia 05.10.24, worked perfectly for 2 months

### ISSUE THREAD

**Issue by Kissao23** [xvzc/SpoofDPI#268]
### Version

Spoof-dpi-v0.12.0

### Operating System

OS Sequoia 15.0.1

### How are you running SpoofDPI?

Just an app, worked perfectly for 2 months
Youtube doesnt work, twitter too, :(

### Description

Stopped working yesterday's evening, nothing helped.

**Comment by EazzyBuzzy** [comment]
Does not work either for me ever since Oct. 5, 2024. Until this day it have worked perfectly for more than a month
Spoof DPI ver.:0.12.0
Mac OS ver.: Sonoma 14.6.1 
Initially installed and then re-installed in order to make it work again (no luck though) via Home Brew. 
Cannot reach out for YouTube, Linkedin, Twitter etc now

**Comment by S4nthetik** [comment]
If doesn't working try to change the dns (1.1.1.1 for example). It worked for me

**Comment by itkandrey** [comment]
macOS: Sonoma Version 14.7
ISP: Rostelecom
 
I confirm that after Oct. 5 SpoofDPI stoped working. Before that everything worked perfectly, although I noticed some delays in video stream transfer speed recently.

I've been trying fixing it since Oct. 5, didn't succeed so far. I tried --window-size 0 (also 1 and 2), --dns-ipv4-only and --dns-addr 1.1.1.1 nothing helped.

<img width="2022" alt="Screenshot 2024-10-06 at 21 43 07 (2)" src="https://github.com/user-attachments/assets/42416e01-5fcd-46b4-8d98-88fbf54819a6">
